<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgAlarmistGreeter</class>
 <widget class="QDialog" name="dlgAlarmistGreeter">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>269</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Greeter</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QFrame" name="frame1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>251</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Garamond</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>W E L C O M E</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMoreoptions">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>More Options</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnContinue">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Click here to boot</string>
    </property>
    <property name="text">
     <string>Continue</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblEyes">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>301</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>scary-eyes-tattoo-on-arm.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>lblEyes</zorder>
   <zorder>label</zorder>
   <zorder>btnMoreoptions</zorder>
   <zorder>btnContinue</zorder>
  </widget>
  <widget class="QFrame" name="frame2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QRadioButton" name="radSpoofYes">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Yes</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radSpoofNo">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>No</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Spoofing MAC addresses hides the serial number of your network cards
to the local network. This can help you hide your geographical location.

It is generally safer to spoof MAC addresses, but it might
also raise suspicions or cause network connection problems.</string>
    </property>
    <property name="text">
     <string>Spoof your MAC address?</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QCheckBox" name="chkAllowrootpassword">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>You have the option of entering an administration password in case you need to perform
administrative tasks. If you choose not to, it will be disabled for better security.</string>
    </property>
    <property name="text">
     <string>Allow
root
pwd</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledPassword1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledPassword2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>This option makes Alarmist look more like Microsoft Windows XP. This
may be ueful in public places in order to avoid attracting suspicion.</string>
    </property>
    <property name="text">
     <string>Camouflage as Windows?</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radCamouflageNo">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>No</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radCamouflageYes">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Yes</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Is your network connection clear of obstacles? If so, and you would like
to connect directly to the Tor network, say (Y)es. On the other hand, If
your computer's network connection is censored, filtered, or proxied, say
(N)o and configure your bridge, firewall, and proxy settings manually.</string>
    </property>
    <property name="text">
     <string>Directly connect to Network?</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radDirectNo">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>No</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radDirectYes">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Yes</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btnMoreoptions</tabstop>
  <tabstop>btnContinue</tabstop>
  <tabstop>radCamouflageYes</tabstop>
  <tabstop>radCamouflageNo</tabstop>
  <tabstop>chkAllowrootpassword</tabstop>
  <tabstop>ledPassword1</tabstop>
  <tabstop>ledPassword2</tabstop>
  <tabstop>radSpoofYes</tabstop>
  <tabstop>radSpoofNo</tabstop>
  <tabstop>radDirectYes</tabstop>
  <tabstop>radDirectNo</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ledPassword1</sender>
   <signal>returnPressed()</signal>
   <receiver>ledPassword2</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>500</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ledPassword2</sender>
   <signal>returnPressed()</signal>
   <receiver>dlgAlarmistGreeter</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>500</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnContinue</sender>
   <signal>clicked()</signal>
   <receiver>dlgAlarmistGreeter</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
